### PR TITLE
TW-903 Add test IDs to some elements

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

**Review by lendihop — APPROVED** [review]

**Review by herkoss — APPROVED** [review]
GJ!

**Review by lendihop — APPROVED** [review]

**Review by herkoss — APPROVED** [review]
GJ!

File: src/components/checkbox/checkbox.tsx [hunk 1]
```diff
@@ -19,14 +22,20 @@ export const Checkbox: FC<CheckboxProps> = ({
   testID
 }) => {
   const colors = useColors();
+  const { trackEvent } = useAnalytics();
+
+  const handlePress = useCallback(() => {
+    trackEvent(testID, AnalyticsEventCategory.ButtonPress, { value: !value });
+    onChange?.(!value);
```
Comment (lendihop) [inline src/components/checkbox/checkbox.tsx:29]: what is this "?." for?

File: src/modals/manage-farming-pool-modal/details-card/index.tsx [hunk 2]
```diff
@@ -112,15 +115,26 @@ export const DetailsCard: FC<DetailsCardProps> = ({
...
+        farmId: farm.id,
+        farmContractAddress: farm.contractAddress
+      };
+
       doAfterConfirmation(
         'Your claimable rewards will be claimed and sent to you. But your full rewards will be totally lost and redistributed among other participants.',
         'Claim rewards',
-        claimRewards
+        () => {
+          trackEvent('CLAIM_REWARDS_MODAL_CONFIRM', AnalyticsEventCategory.ButtonPress, modalAnswerAnalyticsProperties);
```
Comment (lendihop) [inline src/modals/manage-farming-pool-modal/details-card/index.tsx:128]: pls extract selectors into separate enum

File: src/modals/manage-farming-pool-modal/withdraw-form/use-withdraw-formik.ts [hunk 3]
```diff
@@ -77,13 +82,13 @@ export const useWithdrawFormik = (farmId: string, farmVersion: FarmVersionEnum)
             navigateAction(ModalsEnum.Confirmation, {
               type: ConfirmationTypeEnum.InternalOperations,
               opParams,
-              testID: 'STAKE_TRANSACTION_SENT'
+              testID: 'WITHDRAW_TRANSACTION_SENT'
```
Comment (lendihop) [inline src/modals/manage-farming-pool-modal/withdraw-form/use-withdraw-formik.ts:85]: and this selectors also

Comment (keshan3262) [reply src/modals/manage-farming-pool-modal/withdraw-form/use-withdraw-formik.ts:85]: Sorry, test IDs like these have never been specified in enums

File: src/modals/manage-farming-pool-modal/details-card/index.tsx (hunk 2 above)
Comment (keshan3262) [reply src/modals/manage-farming-pool-modal/details-card/index.tsx:128]: Actually, it's not a selector but an event name. Actually, there is no way to assign test IDs to buttons of an alert

**Review by herkoss — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_804

**Review by vladvn1905 — APPROVED** [review]
ios done build : 1.15_804

**Review by lendihop — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]
ios/android done build : 1.15_807

**Pull request merged**